app/models/translation.rb

```diff
@@ -43,12 +43,14 @@ def build_translated_page(page_id, strict)
     xml.to_s
   end
 
-  # TODO: parse JSON here?
   def download_translated_phrases(page_filename)
-    RestClient.get "https://platform.api.onesky.io/1/projects/#{resource.onesky_project_id}/translations",
-                   params: { api_key: ENV['ONESKY_API_KEY'], timestamp: AuthUtil.epoch_time_seconds,
-                             dev_hash: AuthUtil.dev_hash, locale: language.code,
-                             source_file_name: page_filename, export_file_name: page_filename }
+    response = RestClient.get "https://platform.api.onesky.io/1/projects/#{resource.onesky_project_id}/translations",
+                              params: { api_key: ENV['ONESKY_API_KEY'], timestamp: AuthUtil.epoch_time_seconds,
+                                        dev_hash: AuthUtil.dev_hash, locale: language.code,
+                                        source_file_name: page_filename, export_file_name: page_filename }
+    return JSON.parse(response)
+  rescue JSON::ParserError
+    raise Error::PhraseNotFoundError, 'No translated phrases found for this language.'
```

Comment:
This seems like it would be likely to catch other errors that you might be throwing away. Like timeouts or auth errors. Could you check the status code instead?